src/components/MapView/Legends/index.tsx

```diff
@@ -15,32 +14,37 @@ import {
   withStyles,
 } from '@material-ui/core';
 import { Visibility, VisibilityOff } from '@material-ui/icons';
+import React, { PropsWithChildren, useState } from 'react';
 
-import { useSelector } from 'react-redux';
 import { createGetLegendGraphicUrl } from 'prism-common';
-import { Extent } from '../Layers/raster-utils';
-import { mapSelector } from '../../../context/mapStateSlice/selectors';
-import ColorIndicator from './ColorIndicator';
-import LoadingBar from './LoadingBar';
+import { useSelector } from 'react-redux';
 import {
-  LayerType,
   ExposedPopulationDefinition,
+  LayerType,
   LegendDefinitionItem,
 } from '../../../config/types';
 import {
   analysisResultSelector,
   isAnalysisLayerActiveSelector,
 } from '../../../context/analysisResultStateSlice';
+import { mapSelector } from '../../../context/mapStateSlice/selectors';
+import { Extent } from '../Layers/raster-utils';
```

Comment:
I see a few order changes, are we using a different linter? I don't mind too much either way but we should make sure that we use a consistent heuristic to avoid unnecessary line changes

Reply:
I think that this file has been linted for the imports